stripe/src/main/java/com/stripe/android/EphemeralKey.java

```diff
@@ -4,64 +4,19 @@
 import android.os.Parcelable;
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
-import android.support.annotation.VisibleForTesting;
 
-import com.stripe.android.model.StripeJsonModel;
-
-import org.json.JSONArray;
-import org.json.JSONException;
 import org.json.JSONObject;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
-/**
- * Represents an Ephemeral Key that can be used temporarily for certain operations.
- */
-class EphemeralKey extends StripeJsonModel implements Parcelable {
-
-    static final String FIELD_CREATED = "created";
-    static final String FIELD_EXPIRES = "expires";
-    static final String FIELD_SECRET = "secret";
-    static final String FIELD_LIVEMODE = "livemode";
-    static final String FIELD_OBJECT = "object";
-    static final String FIELD_ID = "id";
-    static final String FIELD_ASSOCIATED_OBJECTS = "associated_objects";
-    static final String FIELD_TYPE = "type";
+// Provided for backward compatibility
+// Use CustomerEphemeralKey instead
+@Deprecated
+public class EphemeralKey extends CustomerEphemeralKey {
```

Comment:
Not sure that we really need to keep that class around? It's public, but I fail to see when it would be used by a developer